tests/unittests/test_rpc_messages.py

```diff
@@ -69,3 +69,30 @@ def test_successful_sys_path_import(self):
         self._verify_sys_path_import(
             'success',
             'This module was imported!')
+
+    def _verify_azure_namespace_import(self, result, expected_output):
+        try:
+            path_import_script = os.path.join(
+                testutils.UNIT_TESTS_ROOT,
+                'azure_namespace_import',
+                'test_azure_namespace_import.sh')
+
+            subprocess.run(['chmod +x ' + path_import_script], shell=True)
+
+            output = subprocess.check_output(
+                [path_import_script, result, 'namespace_location_b'],
```

Comment:
Do we also want this to be something like `$SCRIPT_DIR/namespace_location_b` so we don't have to rely on relative paths?